src/styles/styles.js

```diff
@@ -19,6 +19,7 @@ import pointerEventsNone from './pointerEventsNone';
 import pointerEventsAuto from './pointerEventsAuto';
 import overflowXHidden from './overflowXHidden';
 import CONST from '../CONST';
+import emojiHeaderContainerPlatformStyles from './emojiHeaderContainerPlatformStyles';
```

Comment:
Respecting the order of the imports, it would be better to import `emojiHeaderContainerPlatformStyles` right after `optionAlternateTextPlatformStyles`.

Reply:
Applied change